storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/common/JdbcTemplater.java

```diff
@@ -32,12 +33,15 @@ static String getTableName(FeatureSetProto.FeatureSetSpec featureSetSpec) {
         .replaceAll("-", "_");
   }
 
-  String getTableCreationSql(FeatureSetProto.FeatureSetSpec featureSetSpec);
+  static String getTableNameFromFeatureSet(String featureSet) {
+    System.out.println("getTableNameFromFeatureSet--" + featureSet);
```

Comment:
is this the convention or is there a proper logger?

Reply:
I think this is just for our testing purpose, correct me if I'm wrong @swatiarora0208

Reply:
it is not the logger.. removed it